File: Ejercicios iOS/HolaMundo-Objective-C/HolaMundo2/ViewController.m
//
//  ViewController.m
//  HolaMundo2
//
//  Created by Virtualizacion02 on 24/08/23.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    [_etiqueta setText:@""];
    
}

- (IBAction)presionar:(id)sender {
    
    [_etiqueta setText:@"Hola mundo en iOS"];
    
}
@end

File: Ejercicios iOS/HolaMundo-Swift/HolaMundoSwift/ViewController.swift
//
//  ViewController.swift
//  HolaMundoSwift
//
//  Created by Virtualizacion02 on 24/08/23.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var etiqueta: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        etiqueta.text=""
    }


    @IBAction func presionar(_ sender: Any) {
        etiqueta.text="Hola mundo"
    }
}